product/reports/680_Performance by Asset Type - Middleware Datasources/120_Datasource Pool - hourly for the last day.yaml

```diff
@@ -0,0 +1,89 @@
+--- 
+where_clause: 
+dims: 
+created_on: 2016-07-07 14:53:10.822035 Z
+reserved: 
+title: "Datasource Pool - hourly for the last day"
+conditions: 
+updated_on: 2016-07-08 17:23:33.732482 Z
+order: Descending
+graph: 
+menu_name: "Datasource Pool - hourly for the last day"
+rpt_group: Custom
+priority: 
+col_order: 
+- id
+- name
+- start_time
+- mw_ds_available_count_min
+- mw_ds_available_count_avg
+- mw_ds_available_count_max
+- mw_ds_in_use_count_min
+- mw_ds_in_use_count_avg
+- mw_ds_in_use_count_max
+- mw_ds_timed_out_min
+- mw_ds_timed_out_avg
+- mw_ds_timed_out_max
+- mw_ds_average_get_time_min
+- mw_ds_average_get_time_avg
+- mw_ds_average_get_time_max
+- mw_ds_average_creation_time_min
+- mw_ds_average_creation_time_avg
+- mw_ds_average_creation_time_max
+timeline: 
+id: 680
```

Comment:
@jrafanie so how would you advise to proceed? it can be solved here locally by altering the id. but the hardcoded ids in reports are not a middleware specific issue - it's a cross project thing. would you like to see a separate (and general) issue filed for that?